handlers.go

```
@@ -66,28 +65,36 @@ func StartHandler(w http.ResponseWriter, r *http.Request) {
 
 	body, _ := ioutil.ReadAll(r.Body)
```

Do not ignore errors.